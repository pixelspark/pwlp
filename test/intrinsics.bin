1»ÌÝ ÿ…!uÿ…t†!uuÿ…tt†!uuuÿ…ttt†!ãä