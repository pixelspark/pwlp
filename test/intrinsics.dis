0000.	31	PUSHI	[14, bb, cc, dd]
0005.	20	PEEKB	0
0006.	11	PUSHB	[ff]
0008.	85	BINARY	AND
0009.	21	PEEKB	1
0010.	75	UNARY	SHR8
0011.	11	PUSHB	[ff]
0013.	85	BINARY	AND
0014.	74	UNARY	SHL8
0015.	86	BINARY	OR
0016.	21	PEEKB	1
0017.	75	UNARY	SHR8
0018.	75	UNARY	SHR8
0019.	11	PUSHB	[ff]
0021.	85	BINARY	AND
0022.	74	UNARY	SHL8
0023.	74	UNARY	SHL8
0024.	86	BINARY	OR
0025.	21	PEEKB	1
0026.	75	UNARY	SHR8
0027.	75	UNARY	SHR8
0028.	75	UNARY	SHR8
0029.	11	PUSHB	[ff]
0031.	85	BINARY	AND
0032.	74	UNARY	SHL8
0033.	74	UNARY	SHL8
0034.	74	UNARY	SHL8
0035.	86	BINARY	OR
0036.	02	POP	2

