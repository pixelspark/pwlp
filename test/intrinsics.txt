a=irgb(0x03<<1,0xBB,0xCC,0xDD);
b=index(a)|(red(a)<<8)|(green(a)<<16)|(blue(a)<<24);
set_pixel(a);
blit;