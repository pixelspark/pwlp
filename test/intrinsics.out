000000 000000 000000 000000 000000 000000 bbccdd 000000 000000 000000 
